cfy_manager/components/nginx/config/https-file-server.cloudify

```diff
@@ -2,7 +2,6 @@
 server {
     # server listening for internal requests
     listen              127.0.0.1:53229 ssl;
-    listen              [::]:53229 ssl;
```

Comment:
Then it should listen on [::1]:53229 ssl

Reply:
no.